libraries/chain/include/eosio/chain/transaction.hpp

```diff
@@ -127,6 +127,24 @@ namespace eosio { namespace chain {
          return account_name();
       }
 
+      account_name authorizer()const {
```

Comment:
I would change this to something like resource_payer()

Reply:
Done